collect.py

```diff
@@ -92,22 +93,43 @@ def do_something(driver, elem_id=None):
         random.shuffle(children)
 
         for child in children:
-            elem_id = child.get_attribute('id')
 
-            # We need to store the ID in order to replicate what we are doing, so we
-            # have to skip elements with no ID.
-            if elem_id == '':
-                continue
+            # Get all the attributes of the child.
+            child_attributes = driver.execute_script('var elem_attribute = {}; for (i = 0; i < arguments[0].attributes.length; i++) { elem_attribute[arguments[0].attributes[i].name] = arguments[0].attributes[i].value }; return elem_attribute;', child)
```

Comment:
Can you make this script more readable by using a multiline string? Like in `wait_loaded`.

Reply:
It would be great if we could use the Selenium API to do the same, but for now it's fine.

Reply:
@marco-c  I will try to do it with Selenium API.